taca/illumina/MiSeq_Runs.py

```diff
@@ -1,22 +1,12 @@
 import os
-import re
-import csv
-import glob
-import shutil
-import gzip
-import operator
-import subprocess
-from datetime import datetime
-from taca.utils.filesystem import chdir, control_fastq_filename
 from taca.illumina.HiSeq_Runs import HiSeq_Run
```

Comment:
Were you proposing to replace this with HiSeqX_Run when deleting HiSeq_Run?

Reply:
No. What I was proposing was to remove anything to do with the old sequences **except** the HiSeq_Run and HiSeqX_Run classes, since the classes for the new sequences inherit from these. However, at a later stage I plan to have a look at this inheritance since it doesn't make sense to me. Yes, they share functionality, but a MiSeq run is not a subclass of a HiSeq run in my mind. But for now I'm focusing on adding tests and removing unused code to make the porting to py3 easier 🙂

Reply:
Sounds good 👍

Reply:
Maybe move all common functionality to the base Run class?

Reply:
Yeah, something like that would be good 🙂